features/FEATURE_LWIP/lwip-interface/CONTRIBUTING.md

```diff
@@ -0,0 +1,129 @@
+# Description
+
+This document describes how to update mbed OS lwip stack. Mbed OS lwip stack is a copy of
+lwip master repository. Stack is located in `FEATURE_LWIP/lwip-interface/lwip` directory.
+
+When new releases or single commits are added from lwip master repository they need to be
+fetched via mbed OS lwip fork repository. Repository is used to rename lwip source files 
+with `lwip_` prefix to make then compatible with mbed OS build system.
+
+* Lwip master repository is under lwip project (<https://git.savannah.nongnu.org/git/lwip.git>).
+
+* Mbed OS lwip fork repository is under ARMmbed (<https://github.com/ARMmbed/lwip>)
+
+## How to integrate a release from lwip master repository to mbed OS
+
+Integrating a new release is done in following steps.
+
+* Fetch the release to mbed OS lwip fork.
+
+* Prefix files in mbed OS lwip fork with `lwip_` prefix.
+
+* Merge release to mbed OS repository.
+
+### Fetching release to mbed OS lwip fork and prefixing files
+
+1. Clone mbed OS lwip fork repository.
+
+    `git clone git@github.com:ARMmbed/lwip.git`
+
+2. Go to mbed OS lwip fork root directory and add lwip master repository as remote.
+
+    `git remote add <remote name> https://git.savannah.nongnu.org/git/lwip.git`
+
+    `e.g. git remote add lwip https://git.savannah.nongnu.org/git/lwip.git`
+
+3. Pull needed release from lwip master repository using release tag. Tags are found from <https://savannah.nongnu.org/projects/lwip>.
+
+    `git pull <remote name> <tag>`
+
+    `e.g. git pull lwip STABLE-2_0_2_RELEASE_VER`
+
+4. Push tag to mbed OS lwip fork repository to keep fork in sync.
```

Comment:
- Please change "tag" to "the tag".
- Please change "mbed OS lwip" to "the mbed OS LWIP".
- Please change "keep fork" to "keep the fork".